pkgs/development/libraries/science/math/scalapack/default.nix

```diff
@@ -30,6 +30,10 @@ stdenv.mkDerivation rec {
       )
   '';
 
+  # Increase individual test timeout from 1500s to 10000s because hydra's builds
+  # sometimes fail due to this
+  checkFlagsArray = [ "ARGS=--timeout 10000" ];
```

Comment:
The Hydra build timeout is 3600 s?

Reply:
Probably confusing this with @GrahamcOfBorg's timeout, which is 3600, hydra has probably a much longer one. The current build of this package already takes about 1:30h

Reply:
ah ok, you're right.